CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
  - Added support for compiler warnings (#1779, #1905, #1906)
  - Added support for DISABLED flag for database migration scripts (#1913)
  - Added v5 database migration script (#1914)
+ - Check to fail if DB Schema downgrade would be necessary (#1878)
```

Comment:
Isn't this a breaking change? 

I would word it as 
`Server will no longer start if the database schema is for a newer version`